### PR TITLE
Implemeted functionallity to login

### PR DESCRIPTION
**Pull request by TE4-oskar-pilborg**
# Pull Request Template

## What issue did you implement or fix?
Fixed issue #2 

## Summary
- Fixed functionality for login 

## Type of change

- [ ] New feature (non-breaking change which adds functionality)

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules

## How can this implementation be improved?
- You tell me :P

### CODE REVIEW

File: backend/logomatic_backend.rb [hunk 1]
```diff
@@ -53,18 +53,31 @@ def authenticated?
...
 
-    user = @db.execute('SELECT * FROM users WHERE username = ?', user_form['username']).first
-    unless user && BCrypt::Password.new(user['encrypted_password']) == user_form['password']
+    unless user && BCrypt::Password.new(user['hashed_pwd']) == user_form['password']
       [401, ""]
     else
 
       response = {
-        token: JWT.encode({id: user['id']}, MY_SECRET_SIGNING_KEY)
+        token: JWT.encode({id: user['id'], admin: user['admin'], email: user['email'], password: ['hashed_pwd']}, MY_SECRET_SIGNING_KEY)
```
Comment (TE4-jacob-lundberg) [inline backend/logomatic_backend.rb]: Are you sure you need to encode all those attributes with JWT? Should be enough id I think.

File: src/views/LoginView.vue [hunk 2]
```diff
@@ -1,7 +1,5 @@
 <template>
```
Comment (TE4-jacob-lundberg) [inline src/views/LoginView.vue]: Did not receive an explanation as to why you are using a view to only render another component

File: backend/logomatic_backend.rb (hunk 1 above)
Comment (TE4-oskar-pilborg) [reply backend/logomatic_backend.rb]: fixed

File: backend/logomatic_backend.rb [hunk 3]
```diff
@@ -53,18 +53,31 @@ def authenticated?
...
   end
 
-end
\ No newline at end of file
+  #get single user and check admin
+  get 'api/v1/users/:id' do
+    content_type :json
+    user_form = JSON.parse(request.body.read)
+    admin = @db.execute('SELECT admin FROM users WHERE id = ?', user_form['id']).first
+    if admin == 1
```
Comment (TE4-jacob-lundberg) [inline backend/logomatic_backend.rb]: What does this if-clause do? Am I missing something?

**Pull request merged**